benchmarks/benchmarks/sparse_matrix_power.py

```diff
@@ -0,0 +1,19 @@
+from .common import Benchmark, safe_import
+
+with safe_import():
+    from scipy.sparse import random
+
+
+class BenchMatrixPower(Benchmark):
+    params = [
+        [0, 1, 2, 3],
```

Comment:
At the risk of blowing this benchmark up a bit, it might be worthwhile to bump the maximum exponent a bit higher. I can think of at least one real-world use-case for large exponents in a sparse matrix power operation: computing the number of walks in a graph (see e.g. networkx/networkx#5908)

Not a blocker, but adding e.g. 8, 9 to the list might better cover this case!